dist/verbalexpressions.js

```diff
@@ -1,473 +1,599 @@
+         * @param {String} value value to replace with
+         * @return {VerbalExpression} same instace of VerbalExpression
+         */
+
+    }, {
+        key: 'replace',
+        value: function replace(source, value) {
+            source = source.toString();
+            return source.replace(this, value);
         }
-    };
 
-    /**
-    * @return {VerbalExpression} Returns a new instance of VerbalExpressions
-    */
-    function createVerbalExpression() {
-        return new VerbalExpression();
-    }
+        /**
+         * Convert to RegExp object
+         * @return {RegExp} Converted RegExp instance
+         */
+
+    }, {
+        key: 'toRegExp',
+        value: function toRegExp() {
+            var components = this.toString().match(/\/(.*)\/([gimuy]+)?/);
+            var pattern = components[1];
+            var flags = components[2];
 
-    // UMD (Universal Module Definition), URL: https://github.com/umdjs/umd
-    // Supports AMD, CommonJS and the browser
-    if (typeof module !== 'undefined' && module.exports) {
-        // Node.js Module
-        module.exports = createVerbalExpression;
-    } else if (typeof define === 'function' && define.amd) {
-        // AMD Module
-        define(MODULE_NAME, [], function define() {
-            return VerbalExpression;
-        });
+            return new RegExp(pattern, flags);
+        }
+    }], [{
+        key: 'sanitize',
+        value: function sanitize(value) {
+            if (value.source) {
+                return value.source;
+            }
+
+            if (typeof value === 'number') {
+                return value;
+            }
+
+            // Regular expression to match meta characters
+            // https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/regexp
+            var toEscape = /([\].|*?+(){}^$\\:=[])/g;
+
+            // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/lastMatch
+            var lastMatch = '$&';
+
+            // Escape meta characters
+            return value.replace(toEscape, '\\' + lastMatch);
+        }
+    }]);
+
+    return VerbalExpression;
+}(_extendableBuiltin(RegExp));
+
+// UMD (Universal Module Definition)
+// https://github.com/umdjs/umd
+
+
+(function (root, factory) {
+    if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define([], factory);
+    } else if ((typeof module === 'undefined' ? 'undefined' : _typeof(module)) === 'object' && module.exports) {
+        // Node. Does not work with strict CommonJS
+        module.exports = factory();
     } else {
-        // Browser
-        root[MODULE_NAME] = createVerbalExpression;
+        // Browser globals (root is window)
+        root.VerEx = factory();
     }
-}(this));
+})(this, function () {
+    return function () {
+        return new VerbalExpression();
+    };
+});
+//# sourceMappingURL=verbalexpressions.js.map
+}());
```

Comment:
Now there's still an issue with how the IIFE handles `this`. If you run this in your browser:

```
(function() {
  'use strict';
  console.log(this);
}())
```

it will output `undefined`, because the `this` is not bound to the `window`. In the previous implementation it was resolved like this:

```
(function(root) {
  'use strict';
  console.log(root);
})(this)
```

Could you try to use an UMD plugin instead of IIFE plugin?

Reply:
Removed UMD mechanism from `VerbalExpressions.js`, configured `grunt-umd` to run on `dist/verbalexpressions.js` after transpiling, changed `grunt test` tasks and changed the file being tested in `test/tests.js`.